docs/tutorials/executor_forces_tutorial.rst

```diff
@@ -35,97 +38,135 @@ Calling Script
 
 Complete scripts for this example can be found in the forces_simple_ directory.
 
-Let's begin by writing our calling script to parameterize our simulation and
-generation functions and call libEnsemble. Create a Python file called `run_libe_forces.py`
-containing:
+Let's begin by writing our calling script to specify our simulation and
+generation functions and call libEnsemble. Create a Python file called
+`run_libe_forces.py` containing:
 
 .. code-block:: python
     :linenos:
-    :emphasize-lines: 15,19
+    :emphasize-lines: 17,25, 28
 
     #!/usr/bin/env python
     import os
+    import sys
+
     import numpy as np
     from forces_simf import run_forces  # Sim func from current dir
 
-    from libensemble.libE import libE
-    from libensemble.gen_funcs.sampling import uniform_random_sample
-    from libensemble.tools import parse_args, add_unique_random_streams
+    from libensemble import Ensemble
+    from libensemble.alloc_funcs.start_only_persistent import only_persistent_gens as alloc_f
     from libensemble.executors import MPIExecutor
+    from libensemble.gen_funcs.persistent_sampling import persistent_uniform as gen_f
+    from libensemble.specs import AllocSpecs, ExitCriteria, GenSpecs, LibeSpecs, SimSpecs
+
+    if __name__ == "__main__":
 
-    # Parse number of workers, comms type, etc. from arguments
-    nworkers, is_manager, libE_specs, _ = parse_args()
+        # Initialize MPI Executor
+        exctr = MPIExecutor()
 
-    # Initialize MPI Executor instance
-    exctr = MPIExecutor()
+        # Register simulation executable with executor
+        sim_app = os.path.join(os.getcwd(), "../forces_app/forces.x")
 
-    # Register simulation executable with executor
-    sim_app = os.path.join(os.getcwd(), "../forces_app/forces.x")
-    exctr.register_app(full_path=sim_app, app_name="forces")
+        if not os.path.isfile(sim_app):
+            sys.exit("forces.x not found - please build first in ../forces_app dir")
+
+        exctr.register_app(full_path=sim_app, app_name="forces")
+
+        # Parse number of workers, comms type, etc. from arguments
+        ensemble = Ensemble(parse_args=True, executor=exctr)
 
 On line 15, we instantiate our :doc:`MPI Executor<../executor/mpi_executor>`.
 
 Registering an application is as easy as providing the full file-path and giving
 it a memorable name. This Executor will later be used within our simulation
 function to launch the registered app.
 
-Next define the :ref:`sim_specs<datastruct-sim-specs>` and
-:ref:`gen_specs<datastruct-gen-specs>` data structures. Recall that these
-are used to specify to libEnsemble what user functions and input/output fields to
-expect, and also to parameterize function instances without hard-coding:
+On line 22, we initialize the ensemble. The :meth:`parse_args()<tools.parse_args>`
```

Comment:
Perhaps this should link to or describe the `Ensemble` docs `parse_args=True` parameter? Something like:

"The `parse_args=True` parameter instructs the Ensemble class to read `comms` and `nworkers` from the command line."